src/main/java/mat/server/CqlAttributesRemoteCall.java

```diff
@@ -0,0 +1,52 @@
+package mat.server;
+
+import mat.client.shared.CQLConstantContainer;
+import mat.client.shared.MatRuntimeException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientResponseException;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CqlAttributesRemoteCall {
+
+    @Autowired
+    private RestTemplate restTemplate;
+
+    @Value("${mat.fhirMatServices.url}")
+    private String fhirMatServicesUrl;
+
+    private final static String FHIR_MAT_SERVICES_RECOURSE_FOR_ATTRIBUTES = "/find";
+
+    List<String> fhirDataTypeList = new ArrayList<>();
```

Comment:
Services should be stateless. It's basically a static shared field

Reply:
I shouldn’t be declaring it as Static?

Reply:
no! it's a bad practice to have any stateless components/services.

Reply:
Changed it as a local object